pkg/apis/crd/v1alpha1/types.go

```diff
@@ -434,6 +434,11 @@ type NetworkPolicyPeer struct {
 	//  Exact FQDNs, i.e. "google.com", "db-svc.default.svc.cluster.local"
 	//  Wildcard expressions, i.e. "*wayfair.com".
 	FQDN string `json:"fqdn,omitempty"`
+	// Select all Pods with the ServiceAccount matched by this field, as
+	// workloads in AppliedTo/To/From fields.
+	// Cannot be set with any other selector.
+	// +optional
+	ServiceAccount *NamespacedName `json:"serviceAccount,omitempty"`
```

Comment:
Good to see the API is really simple now!

Reply:
Good catch. I slightly prefer to unify them to `NamespacedName` which could be used to refer to other resources. And if we will use verbose API in the future then I guess this generic struct will still be needed.

Reply:
If unifying them sounds good, do you suggest unifying them in this PR or another one?

Reply:
I would suggest another PR.